actors/user/src/main/java/org/sunbird/user/service/FeedService.java

```diff
@@ -0,0 +1,14 @@
+package org.sunbird.user.service;
+
+import org.sunbird.common.models.response.Response;
+import org.sunbird.dto.SearchDTO;
+import org.sunbird.models.user.Feed;
+
+public interface FeedService {
```

Comment:
Interfcae we are starting with I and then name , so follow same

Reply:
done